frontend/src/components/AddGroupForm/AddGroupForm.js

```diff
@@ -21,10 +23,15 @@ import {
 
 const AddGroup = (props) => {
     const { t } = useTranslation('formElements');
-    const { handleSubmit, pristine, onReset, submitting, match, group, initialize } = props;
+    const location = useLocation();
+    const { handleSubmit, pristine, onReset, submitting, group, initialize } = props;
 
     useEffect(() => {
-        if (group && match.url.includes(links.Edit) && !match.url.includes(links.Student)) {
+        if (
+            group &&
+            location.pathname.includes(links.Edit) &&
+            !location.pathname.includes(links.Student)
```

Comment:
we will have all pages to check if url includes edit, delete... . Can we create a helper for this ?